### PR TITLE
A python only class library for programatic html document construction -- first conda upload for both this package and me

### PR DESCRIPTION
**Pull request by areeda**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ n/a] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
        I am the only maintainer
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by areeda** [comment]
@conda-forge/help-python
--
I believe we're ready for review.
I don't know how express how grateful I am for all the good work.

File: .ci_support/osx64.yaml [hunk 1]
```diff
@@ -12,3 +12,5 @@ channel_sources:
   - conda-forge
 target_platform:
   - osx-64
+
+
```
Comment (ocefpaf) [inline .ci_support/osx64.yaml:16]: Can you please revert this change and the removed recipe? Please editing only the recipe you are adding.

Comment (areeda) [reply .ci_support/osx64.yaml:16]: Hi Sorry for the confusion but I'm not sure what to do.
I ran the script `build-locally.py` which deleted 2 directories and modified this file to point to a local /opt path.
I will try starting over.

Comment (areeda) [reply .ci_support/osx64.yaml]: I tried to revert all changes made by `build-locally.py` `.git diff main` now only lists my new recipe.

@conda-forge-admin, please ping https://github.com/orgs/conda-forge/teams/help-python
